documentation/restarting.markdown

```diff
@@ -17,13 +17,29 @@ There are several ways to send a HUP signal to the Puppet Server process, but th
 
     kill -HUP `pgrep -f puppet-server`
 
+Starting in version 2.7.0, you can also reload Puppet Server by running the "reload" action via the operating system's service framework.  This is analogous to sending a hangup signal but with the benefit of having the "reload" command pause until the server has been completely reloaded, similar to how the "restart" command pauses until the service process has been fully restarted.
```

Comment:
@gguillotte I added some more verbiage about the advantages of the "reload" command vs. SIGHUP in 5eb3e88.  Let me know if there are any additional changes that you'd like to see me make before merging this.  Thanks!